public/app/plugins/panel/canvas/module.tsx

```diff
@@ -39,6 +39,14 @@ export const addStandardCanvasEditorOptions = (builder: PanelOptionsEditorBuilde
     editor: PanZoomHelp,
     showIf: (opts) => config.featureToggles.canvasPanelPanZoom && opts.panZoom,
   });
+  builder.addBooleanSwitch({
+    path: 'infinitePan',
+    name: 'Infinite panning',
```

Comment:
Not in love with this naming, open to suggestions 😅

Reply:
I actually think it's pretty declarative and self-explanatory. 👍

Reply:
`addToggle` may be a good option, shorter, cleaner and also self-explanatory 😁